src/reference/sectionA_definitions.inc

```diff
@@ -250,15 +250,15 @@ The following data representation formats are defined for use in this specificat
 
    .. container:: issue-data-repr-basic-real-sign
 
-      2. SHALL, when the basic real number being represented is negative, begin with the Basic Latin hyphen-minus character :unicode:`002D` as the sign indicator.
+      2. MAY begin with a single Basic Latin plus character :unicode:`002B` or a Basic Latin hyphen-minus character :unicode:`002D` as the sign indicator.
```

Comment:
When it's negative it must begin with the - sign?  

- MAY, when the basic real number being represented is positive, begin with the Basic Latin plus character :unicode:`whateveritis` as the sign indicator.
- SHALL, when the basic real number being represented is negative, begin with the Basic Latin hyphen-minus character :unicode:`002D` as the sign indicator.

Reply:
I'm fine with having positive and negative numbers defined outside the spec, in the usual way. It's captured by the first rule (base 10 int representation) I think